google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -711,9 +754,14 @@ def key(self):
         Returns:
             str: key of this resource
         """
-        unique_key = '/'.join([self.parent().type(),
-                               self.parent().key(),
-                               self['constraint']])
+        if 'constraint' not in self._data:
+            unique_key = '/'.join([self.parent().type(),
+                                   self.parent().key(),
+                                   self[0]['constraint']])
```

Comment:
IIRC CAI provides all constraints per resource in a list format, I don't think adding the first constraint `self[0]['constraint']` to the id is a good idea. I would recommend trying to creating multiple constraints on a resource (e.g. project) and inspect the data generated by CAI. if it's only one row, then parent type and parent key is enough to identify the resource.

Reply:
Do you mean creating multiple org policies?

Reply:
org policy constraints.

Reply:
Just verified. Each constraint is retrieved in a row. Added comment in the code.